include/stdint.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _STDINT_H_
+#define _STDINT_H_
+
+#include <ultra64.h>
+
+typedef long intptr_t;
+typedef unsigned long uintptr_t;
```

Comment:
```suggestion
typedef s32 intptr_t;
typedef u32 uintptr_t;
```
Unfortunately IDO cares about `long` vs `int` for matching, so to be completely safe this should mirror the 32-bit types in `ultratypes.h`, at least for now.

Reply:
any reason this is needed on top of ultratypes?

Reply:
![grafik](https://user-images.githubusercontent.com/13141469/112752009-1e0c6e00-8fd1-11eb-9d81-304b610b3365.png)